tools/buildsys/src/builder.rs

```diff
@@ -89,6 +95,19 @@ impl VariantBuilder {
         let version_image = getenv("BUILDSYS_VERSION_IMAGE")?;
         let version_build = getenv("BUILDSYS_VERSION_BUILD")?;
         let output = getenv("BUILDSYS_OUTPUT_DIR")?;
+        let image_format = match image_format {
+            Some(ImageFormat::IMG) | None => String::from("img"),
+            Some(ImageFormat::VMDK) => String::from("vmdk"),
+        };
+        let image_options = match image_options {
+            Some(options_string) => options_string.join(","),
```

Comment:
`image_opts` feels like the wrong abstraction boundary here - it's not consistent with the other handling of low-level details that takes place in `rpm2img`.

Let's hardcode the stream-optimized options in `rpm2img` when building VMDKs. We can name the format `stream-vmdk` if we need to distinguish it from ordinary VMDK for some reason, but I don't think we do.